marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java

```diff
@@ -0,0 +1,68 @@
+package com.marklogic.hub.dhs.installer.deploy;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.security.DeployAmpsCommand;
+import com.marklogic.mgmt.ManageClient;
+import com.marklogic.mgmt.SaveReceipt;
+import com.marklogic.mgmt.resource.ResourceManager;
+import com.marklogic.mgmt.resource.security.AmpManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import org.springframework.web.client.HttpClientErrorException;
+
+/**
+ * This command is used instead of DeployAmpsCommand when a data-hub-security-admin user is deploying
+ * amps. It's because PUT request fail due to https://bugtrack.marklogic.com/55309.
+ */
+public class DeployHubAmpsCommand extends DeployAmpsCommand {
+
+    @Override
+    public boolean cmaShouldBeUsed(CommandContext context) {
+        return false;
+    }
+
+    @Override
+    protected SaveReceipt saveResource(ResourceManager mgr, CommandContext context, String payload) {
+        SaveReceipt receipt = null;
+        ManageClient manageClient = context.getManageClient();
+        try {
+            receipt = super.saveResource(mgr, context, payload);
+        }
+        catch (HttpClientErrorException ex) {
+            try{
```

Comment:
You should just be able to check the status on "ex" to see if it's 403 - e.g. HttpStatus.FORBIDDEN.equals(ex.getStatusCode()) .